mscore/metaedit.cpp

```diff
@@ -120,5 +130,44 @@ void MetaEditDialog::hideEvent(QHideEvent* event)
       QWidget::hideEvent(event);
       }
 
-}
+//---------------------------------------------------------
+//   openFileLocation
+//---------------------------------------------------------
+
+void MetaEditDialog::openFileLocation(const QString& path, QWidget* parent)
+      {
+      const QFileInfo fileInfo(path);
+      QStringList args;
+#if defined(Q_OS_WIN)
+      QStringList param;
+      if (!fileInfo.isDir())
+            param += QLatin1String("/select,");
+      param += QDir::toNativeSeparators(fileInfo.canonicalFilePath());
```

Comment:
I believe that `QDir::toNativeSeparators()` is not needed

Reply:
But I see we're using `QDir::toNativeSeparators()` in several other places, so I guess it is OK to use here too

Reply:
I took this part of the code from [QtCreator's source](https://github.com/qt-creator/qt-creator/blob/master/src/plugins/coreplugin/fileutils.cpp#L67), which was referenced [in this stackoverflow post](https://stackoverflow.com/questions/3490336/how-to-reveal-in-finder-or-show-in-explorer-with-qt)
It had to be adapted, though.

Reply:
Yes, it may not be needed, but surely won't harm.